svelte/fr/tutorial/07-lifecycle/02-ondestroy/app-a/utils.js

```diff
@@ -0,0 +1,9 @@
+import { onDestroy } from 'svelte';
+
+export function onInterval(callback, milliseconds) {
+	const interval = setInterval(callback, milliseconds);
+
+	onDestroy(() => {
+		// Régler la fuite mémoire ici
```

Comment:
```suggestion
		// Règle la fuite mémoire ici
```
je ne sais pas si ce changement est consistant avec les autres tutos mais à l'infinitif, ça fait bizarre je trouve

Reply:
?

```suggestion
		// Réglez la fuite mémoire ici
```